### ISSUE TITLE
Black Flashing screen

### ISSUE THREAD

**Issue by hatat56709** [LukeShortCloud/winesapOS#598]
When I start winesapos and select Linux it goes into a black flashing screen. is there a fix?

**Comment by LukeShortCloud** [comment]
Hey @hatat56709 , thanks for reporting your issue. This sounds like a known issue in winesapOS 3.2.1 that will be resolved in our upcoming 3.3.0 release. The workaround is to open up "Konsole" and then run these two commands:

```
sudo pacman -S -y -y
sudo pacman -S mesa-steamos mesa-vdpau-steamos opencl-mesa-steamos vulkan-intel-steamos vulkan-mesa-layers-steamos vulkan-radeon-steamos vulkan-swrast-steamos lib32-libva-mesa-driver-steamos lib32-mesa-steamos lib32-mesa-vdpau-steamos lib32-opencl-mesa-steamos lib32-vulkan-intel-steamos lib32-vulkan-mesa-layers-steamos lib32-vulkan-radeon-steamos lib32-vulkan-swrast-steamos libva-mesa-driver-steamos
```

If you cannot do that because of the flickering screen, you can access the console by pressing `CTRL`, `ALT`, and `F4`. Then, log in as the `root` user (the password is also `root`). Run the commands from there. You will need Internet access for these packages to be downloaded.

Let me know if this works or not!

**Comment by LukeShortCloud** [comment]
@hatat56709 winesapOS 3.3.0 has been released to address the issues you were encountering!

https://github.com/LukeShortCloud/winesapOS/releases/tag/3.3.0
